Packs/RecordedFuture/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Recorded Future Intelligence",
     "description": "Recorded Future App, this pack is previously known as 'RecordedFuture v2'",
     "support": "partner",
-    "currentVersion": "1.3.3",
+    "currentVersion": "1.3.4",
```

Comment:
I think that adding a new collector to the pack should be more than a revision, have you asked the tpm/pm/ Dean ?

Reply:
Regarding the new/existing pack question.
There is no way to add an integration with XSOAR support to a pack with partner support
I am waiting for a reply from @DanilVil.
So let's wait for Danil's reply (maybe we should move the integration to a different pack).